src/Resources/views/Core/tab_menu_template.html.twig

```diff
@@ -119,8 +119,9 @@
 {% endblock %}
 
 {% block label %}
-{{-
-    item.label|trans(
+    {# We use method accessor instead of ".label" since `item` implements `ArrayAccess` and could have a property called "label".#}
```

Comment:
```suggestion
    {# We use method accessor instead of ".label" since `item` implements `ArrayAccess` and could have a property called "label". #}
```